mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -432,6 +439,8 @@ The optional `clone` block supports:
 
     A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
 
+* `preferred_zone` - (Optional) (Point-in-time recovery for PostgreSQL only) Clone to an instance in the specified zone. If no zone is specified, clone to the same zone as the source instance.
```

Comment:
Can you also add examples for these fields to the section https://github.com/GoogleCloudPlatform/magic-modules/blob/7d903c9987cf20617ae86f40b54e75ef64dc7c4d/mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown?plain=1#L21 ?

Reply:
Added an example of Enterprise edition with data_cache_config.